packages/webcomponents/src/utils/utils.ts

```diff
@@ -83,16 +89,49 @@ export function formatAddress(address: Address): string {
   return `${address.line1}, ${address.city}, ${address.state} ${address.postal_code}`;
 }
 
-export function extractComputedFontsToLoad() {
-  const computedStyles = getComputedStyle(document.body);
-  return (
-    computedStyles
-      ?.getPropertyValue('--jfi-load-google-font')
-      ?.trim()
-      .replace(/'|"/g, '')
-      .replace(' ', '+') || null
-  );
-}
+export const MapPaymentStatusToBadge = (status: string) => {
```

Comment:
I think this should go somewhere else, like in a payments specific file. That may have even been done already in another branch... may want to check with @colganmeanor

Reply:
This mapping function was already here before this PR.
I think if this was not moved to another file yet we can do it in another PR

Reply:
@TiagoFuelber @jakemer good call - 

I did update this in `main` actually when we merged in the Terminals List component into main. And I believe that update has since been merged back into `release-5.x.x` so this shouldn't be an issue anymore as long as this branch is up to date with `release-5.x.x` ? 🤔